vmm/src/lib.rs

```diff
@@ -129,6 +129,62 @@ impl AsRawFd for EpollContext {
     }
 }
 
+pub fn start_vmm_thread(
+    api_event: EventFd,
+    api_receiver: Receiver<ApiRequest>,
+) -> thread::JoinHandle<()> {
+    thread::Builder::new()
+        .name("cloud-hypervisor-vmm".to_string())
```

Comment:
Might be too long. Check the output in ps.

Reply:
Changed to `vmm`. We know it's a cloud-hypervisor thread anyway.